android/engine/src/main/java/org/smartregister/fhircore/engine/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -116,11 +118,30 @@ open class QuestionnaireViewModel(application: Application) : AndroidViewModel(a
             questionnaire.useContext.filter { it.hasValueCodeableConcept() }.forEach {
               it.valueCodeableConcept.coding.forEach { bun.resource.meta.addTag(it) }
             }
+
+            // add managing organization of logged in user to record
+            (getApplication<Application>() as ConfigurableApplication).authenticatedUserInfo
+              ?.organization?.let { org ->
+              val organizationRef = Reference().apply { reference = "Organization/$org" }
+              val resource = bun.resource
```

Comment:
Can we use full name here instead of **bun** ? e.g. use `bundleEntry`

Reply:
This is array variable. We can change it to 'b' OR 'it'. It was named bun for better readability as it was used in multiple places